### PR TITLE
Logitech.SetPoint version 6.90.66

### PR DESCRIPTION
**Pull request by KasperLK**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102784)

### CODE REVIEW

**Comment by KasperLK** [comment]
@microsoft-github-policy-service agree

**Comment by KasperLK** [comment]
This is the link to Logitech's Support page for SetPoint.
https://support.logi.com/hc/en-us/articles/360025141274

Also https://support.logitech.com/home automatically redirect to https://support.logi.com/hc/en-us

File: manifests/l/Logitech/SetPoint/6.90.66/Logitech.SetPoint.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Logitech.SetPoint
+PackageVersion: 6.90.66
+PackageLocale: en-US
+Publisher: Logitech
+PackageName: Logitech SetPoint
```
Comment (stephengillie) [inline manifests/l/Logitech/SetPoint/6.90.66/Logitech.SetPoint.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.logitech.com/

PrivacyUrl:   http://www.logitech.com/footer/privacy
PackageName: Logitech SetPoint
PackageUrl: https://support.logi.com/hc/en-us/articles/360025141274-SetPoint
```
```suggestion
PublisherUrl: https://www.logitech.com/
PrivacyUrl:   http://www.logitech.com/footer/privacy
PackageName: Logitech SetPoint
PackageUrl: https://support.logi.com/hc/en-us/articles/360025141274-SetPoint
```

Comment (stephengillie) [inline manifests/l/Logitech/SetPoint/6.90.66/Logitech.SetPoint.locale.en-US.yaml:8]:
```suggestion
PublisherUrl: https://www.logitech.com/
PrivacyUrl:   http://www.logitech.com/footer/privacy
PackageName: Logitech SetPoint
PackageUrl: https://support.logi.com/hc/en-us/articles/360025141274-SetPoint
```

**Review by stephengillie — DISMISSED** [review]

**Comment by KasperLK** [comment]
I have no idea how to solve this.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**